src/logic/operate.js

```diff
@@ -0,0 +1,30 @@
+import Big from 'big.js'; // eslint-disable-line
+
+const operate = (numberOne, numberTwo, operand) => {
+  const numOne = Big(numberOne);
+  const numTwo = Big(numberTwo);
+  let result;
+  switch (operand) {
+    case '+':
+      // .plus
+      result = numOne.plus(numTwo).toString();
+      break;
+    case '-':
+      // .minus
+      result = numOne.minus(numTwo).toString();
+      break;
+    case '*':
+      // .times
+      result = numOne.times(numTwo).toString();
+      break;
+    case '÷':
+      // .div
+      result = numOne.div(numTwo).toString();
+      break;
```

Comment:
Dont forget to handle division by ZERO on the next milestone

Reply:
thanks for the tip :+1: